src/repositories/indexable-repository.php

```diff
@@ -149,4 +149,15 @@ public function create_for_id_and_type( $object_id, $object_type ) {
 
 		return $indexable;
 	}
+
+	/**
+	 * Retrieves indexables by their IDs.
+	 *
+	 * @param array $indexable_ids The indexable IDs.
+	 *
+	 * @return array|\IdiormResultSet Instances of indexables.
+	 */
+	public function find_by_indexable_ids( $indexable_ids ) {
```

Comment:
You can typehint the `$indexable_ids` to make sure the given value is an array 
```suggestion
	public function find_by_indexable_ids( array $indexable_ids ) {
```

Reply:
Thanks for the tip! :)